docs/source/index.rst

```diff
@@ -109,45 +69,13 @@ Welcome to TensorRT-LLM's Documentation!
    advanced/speculative-decoding.md
    advanced/disaggregated-service.md
 
-.. toctree::
-   :maxdepth: 2
-   :caption: Performance
-   :name: Performance
-
-   performance/perf-overview.md
-   Benchmarking <performance/perf-benchmarking.md>
-   performance/performance-tuning-guide/index
-   performance/perf-analysis.md
-
-
-.. toctree::
-   :maxdepth: 2
-   :caption: Reference
-   :name: Reference
-
-   reference/troubleshooting.md
-   reference/support-matrix.md
-
-   .. reference/upgrading.md
-
-   reference/precision.md
-   reference/memory.md
-   reference/ci-overview.md
-
 
 .. toctree::
    :maxdepth: 2
-   :caption: Blogs
+   :caption: Links
```

Comment:
suggestion: "Project Links" or "Project Resources"

Just a nit that maybe "Links" is a little vague, so I was trying to think of something more descriptive.

Reply:
would "References" or "Quick Links" work?